tests/test_date.py

```diff
@@ -2,90 +2,158 @@
 # -*- coding: utf-8 -*-
 from __future__ import unicode_literals
 
+from types import MethodType
+
+from collections import OrderedDict
+
```

Comment:
It helps to re-familiarize yourself with PEP-8 once in a while. I myself do this about once a year.
Please refresh your memory with [this section](https://www.python.org/dev/peps/pep-0008/#imports) or whole document.